dvc/cli.py

```diff
@@ -84,6 +85,26 @@
 ]
 
 
+def _find_cmd_suggestions(cmd_arg, cmd_choices):
+    """Find similar command suggestions for a typed command that contains typos.
+
+    Args:
+        cmd_arg: command argument typed in.
+        cmd_choices: list of valid dvc commands to match against.
+
+    Returns:
+        String with command suggestions to display to the user if any exist.
+    """
+    suggestions = get_close_matches(cmd_arg, cmd_choices)
+
+    suggestion_str = ""
```

Comment:
So right now, (once the `sys.argv` issue is resolved), we will just exit with the `most similar commands are...` message, but don't actually note that the problem is due to an invalid command. I think we should start `suggestion_str` with a line similar to git's:

```
git: 'commti' is not a git command. See 'git --help'.
```